ckanext/datagovcatalog/helpers/sitemap.py

```diff
@@ -4,8 +4,8 @@
 
 def get_sitemap_url():
     s3_url = config.get('ckanext.geodatagov.s3sitemap.aws_s3_url', 'https://filestore.data.gov/')
-    s3_path = config.get('ckanext.geodatagov.s3sitemap.aws_storage_path', 'gsa/catalog-next/sitemap')
+    s3_path = config.get('ckanext.geodatagov.s3sitemap.aws_storage_path', 'gsa/catalog-next/sitemap/')
```

Comment:
Just thinking about this again, and it seems a little strange to be consuming a config option from another plugin, especially since there's no explicit dependency on ckanext-geodatagov. Here we're actually creating coupling on geodatagov by using some internal assumptions a) two config fields and how they work b) the sitemap filename is sitemap.xml.

Maybe we should be duplicating the config setting as `ckanext.datagovcatalog.sitemap_url`. Then it could be a complete URL instead of concatenating three strings and really de-couples this extension from the sitemap generation.